examples/elbv2/lbtargetgroupattachment.yaml

```
@@ -18,7 +18,7 @@ spec:
 apiVersion: elbv2.aws.upbound.io/v1beta1
 kind: LBTargetGroupAttachment
 metadata:
-  name: test
+  name: test-instance
```

Could you please add the `meta.upbound.io/example-id` annotation to the manifest:
```suggestion
  name: test-instance
  annotations:
    meta.upbound.io/example-id: elbv2/v1beta1/lbtargetgroupattachment
```